CMakeLists.txt

```diff
@@ -292,7 +296,16 @@ ENDIF()
 # GLES1 and GLES2 libraries are compatible and can be co-linked.
 IF(FREEGLUT_GLES)
     LIST(APPEND PUBLIC_DEFINITIONS -DFREEGLUT_GLES)
-    LIST(APPEND LIBS GLESv2 GLESv1_CM EGL)
+    if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.27")
```

Comment:
Unfortunately it appears that cmake 3.1 doesn't have `VERSION_GREATER_EQUAL`. I guess possible fixes are to either reverse the logic and use `VERSION_LESS`, or use an *and* of separate greater and equal tests.

Reply:
I've inverted the logic to use `NOT` and `VERSION_LESS`.